src/Stepper/StepperHeader.jsx

```diff
@@ -9,29 +9,34 @@ function StepListSeparator() {
   return <li aria-hidden="true" className="pgn__stepper-header-line" />;
 }
 
-function StepList({ steps, activeKey }) {
+function StepList({ steps, activeKey, isClickable }) {
```

Comment:
I'm not sure about the naming. Because the `isClickable` prop seems to be a bool value.
Maybe it's better to change to `clickHandler` or something similar?

Reply:
Yes, you are right
Thanks